models/stripe_reports/stripe__payout_itemized_3.sql

```diff
@@ -0,0 +1,32 @@
+with payout_enhanced as (
+
+    select *
+    from {{ ref('int_stripe__payout_enhanced')}}
+
+)
+
+select
+    payout_id,
+    case 
+        when is_automatic = true
+        then payout_arrival_date_at 
+        else payout_created_at
+    end as effective_at,
```

Comment:
Request to reform case statement to be in line with our [style guidelines](https://github.com/fivetran/dbt_style_guide/blob/main/sql_guidelines.md).
```suggestion
    case 
        when is_automatic = true then payout_arrival_date_at 
        else payout_created_at
    end as effective_at,
```